src/commons/assessmentWorkspace/AssessmentWorkspace.tsx

```diff
@@ -244,7 +245,7 @@ const AssessmentWorkspace: React.FC<AssessmentWorkspaceProps> = props => {
   useEffect(() => {
     if (!selectedTab) return;
 
-    if (!isMobileBreakpoint && mobileOnlyTabIds.includes(selectedTab)) {
+    if ((!isMobileBreakpoint || isVscode) && mobileOnlyTabIds.includes(selectedTab)) {
       setSelectedTab(SideContentType.questionOverview);
     }
   }, [isMobileBreakpoint, props, selectedTab, setSelectedTab]);
```

Comment:
Can double check the dependency array?

Reply:
<img width="800" alt="image" src="https://github.com/user-attachments/assets/f9742778-85f5-449b-8521-230c34da6a32" />

Reply:
Added to deps array